tests/unit/src/Internal/Payment/PaymentRequestTest.php

```diff
@@ -21,12 +27,17 @@ class PaymentRequestTest extends WCPAY_UnitTestCase {
 	 */
 	private $sut;
 
+	public function setUp(): void {
+		parent::setUp();
+		$this->mock_legacy_proxy = $this->createMock( LegacyProxy::class );
```

Comment:
Oh, I forgot about this, but had it as a local change, and remembered now: Let's define the `$mock_legacy_proxy` var with type `LegacyProxy|MockObject`. Not a problem functionally, but my editor raises too many alarms :D

Reply:
Good point, I would also have declared property `$this->mock_legacy_proxy` too. Addressed  6aeaf6161907d1bde56b6a2604c83ab22ede4e9e